packages/main/src/Option.ts

```diff
@@ -58,7 +58,7 @@ class Option extends ListItemBase implements IOption {
 	 * @public
 	 */
 	@property()
-	value?: string;
```

Comment:
We need to think about this because if we have following case:

```
<ui5-select required>
   <ui5-option>Item 1</ui5-option> // "Item 1" should be the value of select
   <ui5-option value="">Item 2</ui5-option> // should trigger required state
   <ui5-option value="item3_value">Item 3</ui5-option> // "item3_value" should be the value of select
</ui5-select>
```

Reply:
As we synced offline, lets keep value here to accept undefined too. Otherwise the required property will be redundant in the example provided earlier.